### DISCUSSION TITLE
Need Help Deploying!

### DISCUSSION THREAD

**Discussion by MovieHubX** [vercel/community#1709] [Help]
### Summary

I am trying to deploy my react project in vercel, but i don't know how to run the comand npm install  react-helmet while deploying the project,  there are only theese options!

I would really appreciate if anyone could help! I am trying to figure it out from past 2 hours.
(btw i am not a developer i am learning so idk these stuff properly) 
😃
![image](https://user-images.githubusercontent.com/126613066/223193957-1f5d357a-4486-4dcb-b3cd-fabb5968e59c.png)

**Comment by amyegan** [comment]
Hi @MovieHubX! I want to make sure I get you the right answer, so I don't want to make any assumptions. Can you tell me what steps you have already taken? It would be helpful for me to know if your project came from one of the available [Vercel templates](https://vercel.com/templates) or a different source. If your code repository is public, please share a link here so I can suggest a solution specific to your project
